docs/docusaurus.config.js

```diff
@@ -71,6 +53,11 @@ module.exports = {
   organizationName: 'RasaHQ',
   projectName: 'rasa',
   themeConfig: {
+    algolia: {
+      apiKey: '09ef5d111ebd9002df72575d516693af',
+      indexName: 'BH4D9OD16A',
+      // searchParameters: {}, // Optional (if provided by Algolia)
+    },
```

Comment:
Are we merging the search while its still erroring out? When do we think that will be configured?

Reply:
I'm going to try to disable the form for now, and if that doesn't work I guess we'll leave it out. I had to insert some data here to get it to show up, because of the way Docusaurus validates its config internally